src/Symfony/Component/Validator/Constraints/Image.php

```diff
@@ -72,6 +78,8 @@ class Image extends File
     public $minWidthMessage = 'The image width is too small ({{ width }}px). Minimum width expected is {{ min_width }}px.';
     public $maxHeightMessage = 'The image height is too big ({{ height }}px). Allowed maximum height is {{ max_height }}px.';
     public $minHeightMessage = 'The image height is too small ({{ height }}px). Minimum height expected is {{ min_height }}px.';
+    public $minPixelsMessage = 'The image has to few pixels ({{ pixels }} pixels). Minimum amount expected is {{ min_pixels }} pixels.';
```

Comment:
> What can be done is that al least the height and width parameters will be set, so custom messages can use these.

Makes sense :+1: no strong opinion about the default message (just proposed an alternative). Personally i'd favor `300 × 250` vs. `75000 pixels`